Project.toml

```
@@ -29,6 +29,7 @@ MacroTools = "0.5"
 NaNMath = "0.3, 1"
 SpecialFunctions = "0.8, 0.9, 0.10, 1.0, 2"
 StaticArrays = "0.10, 0.11, 0.12, 1.0"
+Statistics = "1"
```

Based on https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958:
```suggestion
Statistics = "<0.0.1, 1"
```